docker/Dockerfile

```diff
@@ -184,3 +184,14 @@ alias code='code --user-data-dir /root/.visual_code/'\n\
 alias STRESS='stress -c $(nproc) -m $(nproc) -i $(nproc) -d $(nproc)'\n\
 source ${ROS_WS}/devel/setup.bash\n\
 " >> ~/.bashrc
+
+# ===================================================
+# Clone relevant repositories for the vision-project interaction
+# ===================================================
+FROM vision_project_dev as vision_project_interaction
+
+RUN \
+	cd ${ROS_WS}/src \
+	&& git clone https://github.com/emilyxzhou/ros-vision-interaction.git \
```

Comment:
It would be great to add this repository to `robotpt` and maybe this project. We can do that after shipping the demo to Prof. Bittner.